docs/api/target.rst

```diff
@@ -6,23 +6,39 @@
 Target Macros
 =============
 
+.. _blt_add_benchmark:
+
 blt_add_benchmark
 ~~~~~~~~~~~~~~~~~
 
 .. code-block:: cmake
 
-    blt_add_benchmark( NAME [name]
-                       COMMAND [command])
+    blt_add_benchmark( NAME          [name]
+                       COMMAND       [command]
+                       NUM_MPI_TASKS [n])
+
+Adds a benchmark to the project.
+
+NAME
+  Name that CTest reports.
 
-Adds a (google) benchmark test to the project.
+COMMAND
+  Command line that will be used to run the test and can include arguments.  
 
-NAME is used for the name that CTest reports and should include the string 'benchmark'.
+NUM_MPI_TASKS
+  Indicates this is an MPI test and how many MPI tasks to use.
 
-COMMAND is the command line that will be used to run the test and can include arguments.  
+This macro adds the benchmark to the ``run_benchmarks`` build target.
```

Comment:
It might be slightly clearer to say that it add a benchmark test to the ``Benchmark`` ctest configuration, and that benchmarks can be run using the ``run_benchmarks`` build target.

(perhaps this is covered by the `note` though)